Source/WarpX.cpp

```diff
@@ -1413,3 +1422,4 @@ WarpX::PicsarVersion ()
     return std::string("Unknown");
 #endif
 }
+
```

Comment:
Only a single newline at the end of files needed:
```suggestion
```

Reply:
Removed.